README.md

```diff
@@ -110,86 +110,122 @@ bash test/test_cleanup.sh
 
 ## Usage
 
-Now that your virtual environment is set up and the workflows are deployed and
-tested, you can go ahead and run the workflows on your samples.
+Now that your virtual environment is set up and the workflow is deployed and
+tested, you can go ahead and run the workflow on your samples.
 
-But first, here is a brief description of what each of the three workflows
-does:
+But first, here is a brief description of what the workflow does:
 
 ### Workflow description
 
-The repository contains the following workflows, all implemented in Snakemake
-and fully containerized:
+The workflow consist on one main `Snakefile` 
+gathering three different subworkflows,
+**_PREPARE_**, **_MAP_** and **_QUANTIFY_**. 
+Each one of them is implemented in an individual Snakemake file
+and fully containerized.    
+
 
 #### _PREPARE_
 
-The first workflow, **_PREPARE_** downloads and processes "genome resources"
+The first subworkflow, **_PREPARE_**, downloads and processes "genome resources"
```

Comment:
Well, it does _not_ download anything after #13 is merged - and I thought the idea was to merge #13 and #14 first. We _can_ also merge this one first, and then you update the documentation again in #13 and #14. Your call. The important thing is that after merging _this_ PR, it has to be in sync with what is in the default channel, and everything coming in after that has to take care of its own changes to the doc, if necessary.